platforms/hyperledger-besu/charts/node_validator/values.yaml

```diff
@@ -159,3 +159,7 @@ labels:
   service: []
   pvc: []
   deployment: []
+  
+metrics:
```

Comment:
Please provide example values for each k:v pair (enabled, port) (check other key:value pairs in this values file)

Reply:
done